docroot/modules/contrib/custom_solr_search/custom_solr_search.services.yml

```diff
@@ -14,9 +14,6 @@ services:
     class: Drupal\custom_solr_search\FilterQuerySettings
     arguments: []
   custom_solr_search.facet:
-    class: Drupal\custom_solr_search\Facet
-    arguments: []
-  custom_solr_search.facet_config:
```

Comment:
@adityaanurag Are we sure this service is not being used anywhere in the codebase & won't break something thats working.